### PR TITLE
Partially resolves the issue #680

### PR DESCRIPTION
**Pull request by Phantsure**
## Description

Made some changes to formatting and docs to match with numpydocs format.

## Changes made to files

- [x] tardis.plasma.base
- [x] tardis.plasma.properties.base
- [x] tardis.montecarlo.base

### CODE REVIEW

**Comment by Phantsure** [comment]
@jaladh-singhal Yeah, you are right. I was using Pycharm, didn't notice the changes it made. Should i change the name or change the files to match issue? Because I suppose PEP8 guidelines if followed makes a good code, right?

**Comment by Phantsure** [comment]
I removed most PEP8 changes and a few unnecessary import statements too

File: tardis/plasma/base.py [hunk 1]
```diff
@@ -58,8 +56,10 @@ def _build_graph(self):
         """
         Builds the directed Graph using network X
 
-        :param plasma_modules:
-        :return:
+        Parameters
+        ----------
+        ~plasma_properties : list
```
Comment (jaladh-singhal) [inline tardis/plasma/base.py]: Why is `~` needed here?

File: tardis/montecarlo/base.py [hunk 2]
```diff
@@ -197,11 +197,17 @@ def run(self, model, plasma, no_of_packets,
         Parameters
         ----------
         model : tardis.model.Radial1DModel
+            # UNSURE of description of below params
```
Comment (jaladh-singhal) [inline tardis/montecarlo/base.py]: If you don't know about this, better leave it as it is - no need of comment. Similarly, remove `.` from following parameters.

File: tardis/plasma/base.py (hunk 1 above)
Comment (Phantsure) [reply tardis/plasma/base.py]: I thought it is used for parameters that are not passed as arguments, but i didn't find any docs to support that. Should i remove it?

File: tardis/montecarlo/base.py (hunk 2 above)
Comment (Phantsure) [reply tardis/montecarlo/base.py]: okay

**Comment by Phantsure** [comment]
done the changes. will keep on updating/adding docs to other files.

**Pull request closed without merging**